### PR TITLE
Updating Task Owner for Azure pipeline tasks

### PR DESCRIPTION
**Pull request by v-jkarri**
**Task name**: update CODEOWNERS

**Description**: updated owner alias for azure pipeline tasks.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -131,13 +131,13 @@ Tasks/Common/VstsAzureRestHelpers_/  @Ajay-MS  @rajatagrawal-dev
...
 
-Tasks/Common/azure-arm-rest-v2/   @nadesu    @chshrikh     @bishal-pdmsft
+Tasks/Common/azure-arm-rest-v2/   @nadesu @v-jkarri   @chshrikh     @bishal-pdmsft
 
 Tasks/Common/azure-blobstorage-artifactProvider/   @omeshp   @hiyadav    @bishal-pdmsft
 
 Tasks/Common/codeanalysis-common/    @microsoft/akvelon-build-task-team
 
-Tasks/Common/coveragepublisher/     @nadesu   @ganesp
+Tasks/Common/coveragepublisher/     @nadesu @v-jkarri  @ganesp
```
Comment (chshrikh) [inline .github/CODEOWNERS:140]: 'coveragepublisher' is not own by environment, you can remove your name from here

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -173,7 +173,7 @@ Tasks/DelayV1/     @rohit-batra  @naga-bandi
 
 Tasks/DeleteFilesV1/   @microsoft/akvelon-build-task-team
 
-Tasks/DeployVisualStudioTestAgentV2/    @nadesu  @ankit-arora12
+Tasks/DeployVisualStudioTestAgentV2/    @nadesu @v-jkarri @ankit-arora12
```
Comment (chshrikh) [inline .github/CODEOWNERS:176]: DeployVisualStudioTestAgentV2 not own by environment team, you can remove your name

File: .github/CODEOWNERS (hunk 1 above)
Comment (v-jkarri) [reply .github/CODEOWNERS]: Removed my name.

File: .github/CODEOWNERS (hunk 2 above)
Comment (v-jkarri) [reply .github/CODEOWNERS]: Removed my name from DeployVisualStudioTestAgentV2 .

**Review by chshrikh — APPROVED** [review]

**Pull request merged**